about.html

```diff
@@ -0,0 +1,136 @@
+    </header>    
+    <section class="about">
+        <nav class="header_menu">
+            <ul class="Desk_nav">
+              <li class="Desk_list"><a href="./index.html"> Home </a></li>
+              <li class="Desk_list"><a href="#work"> Program </a></li>
+              <li class="Desk_list"><a href="#contact-form"> Join </a></li>
+              <li class="Desk_list"><a href="#work"> Sponsor </a></li>
+              <li class="Desk_list"><a href="#contact-form"> News </a></li>
+            </ul>
+          </nav>
+          <div class="humburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+      <p class="about-para1">"Hello! Great People"</p>
+      <h1>
+        Timbuktu Technical <br />
+        Institute Calling 2023
+      </h1>
+      <hr>
+      <p class="about-para">
+        Timbuktu Technical Institute was form in February 1984 by Nicholas
+        Kwamena Amissah.He believes that with the right form of education, if it
+        is accessible for all, will help accelerate the development of the
+        African Continent. Timbuktu was the worlds first center of education
+        created by the Old Mali Empire. It in in this regard that he chose the
+        name Timbuktu to represent the return to the old African way of doing
+        things.We at Timbuktu Technical Institute believe that every single
+        person is unique and capable that is the main reason why we have come up
+        with this course for free.
+      </p>
+      <p class="about-para2">
+        Please contact us per email for any further question about Timbuktu
+        Technical Institute 2023<br /><b>nkamissah002@st.ug.edu.gh</b></p>
+      </p>
+    </section>
+    <section class="Logo">
+        <h2>
+           Creation of Timbuktu Technical Institution Logo
+        </h2>
+        <hr>
+        <p>The making of the Timbuktu Technical Institution Logo was inspired by the structures of the old Mali empire. The green color represent the beauty, vegetation and the wealth of the African Continent. The motto "The Future, Today", is borne from the fact that when we all come together, we can create the very future we want for ourselves today.</p>
+        <img src="./Images/timbuktu.png" alt="Timbuktu Image">
+    </section>
+    <section class="past-events">
+        <h2>See The Past TTI Global Events</h2>
+        <hr>
+        <p>Take a look at our Timbuktu Technical Institute program that took place in Accra, Ghana in 2022</p>
+        <div>
+          <div class="P-img1">
+            <div class="blur"></div>
+            <h3>Accra, Ghana 2021</h3>
+            <p>This program happened in the capital of Ghana</p>
+          </div>
+            
+          <div class="P-img2">
+            <div class="blur"></div>
+            <h3>Dakar, Senegal 2021</h3>
+            <p>This program happened in the capital of Ghana</p>
+          </div>
+        </div>
```

Comment:
- [OPTIONAL] You did a great job adding the 'orange' overlay over the image, and it would be even better if the text over the image is centralized like the template.

Reply:
I have centered the text
Thank you.